modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -230,21 +244,32 @@ class ConnectionAvgStats {
 
                 this._avgRTT.appendReport(batchReport, props);
 
-                // Report end to end RTT only for JVB
-                if (!this.isP2P) {
-                    const avgRemoteRTT = this._calculateAvgRemoteRTT();
-                    const avgLocalRTT = this._avgRTT.calculate();
+                if (this.isP2P) {
+                    // Report RTT diff only for P2P.
+                    const jvbEnd2EndRTT = this._conference.avgRtpStatsReporter
+                        .jvbStatsMonitor._avgEnd2EndRTT;
 
-                    if (!isNaN(avgLocalRTT) && !isNaN(avgRemoteRTT)) {
-                        // eslint-disable-next-line camelcase
-                        const reportValue = {
-                            value: avgLocalRTT + avgRemoteRTT
+                    if (!isNaN(jvbEnd2EndRTT)) {
+                        const avgRTTDiff
+                            = this._avgRTT.latestAvg - jvbEnd2EndRTT;
+
+                        // eslint-disable-next-line dot-notation
+                        batchReport['stat_avg_rtt_diff'] = {
+                            value: avgRTTDiff
                         };
+                    }
+                } else {
+                    // Report end to end RTT only for JVB.
+                    const avgRemoteRTT = this._calculateAvgRemoteRTT();
+                    const avgLocalRTT = this._avgRTT.calculate();
 
-                        Object.assign(reportValue, props);
+                    this._avgEnd2EndRTT = avgLocalRTT + avgRemoteRTT;
```

Comment:
It's a little bit tricky because it's populated by the `jvbStatsMonitor` but it's accessed by the `p2pStatsMonitor`.